react/features/base/connection/actions.native.ts

```diff
@@ -202,3 +203,20 @@ export function disconnect(_?: boolean) {
         return promise;
     };
 }
+
+// eslint-disable-next-line valid-jsdoc
+/**
+ * Connection indicator visibility.
+ *
+ * @param {Boolean} visible - Connection indicator visibility.
+ * @returns {{
+ *     type: CONNECTION_INDICATOR_VISIBILITY
+ *     visible
+ * }}
+ */
+export function connectionIndicatorVisibility(visible: boolean) {
```

Comment:
This probably belongs in the filmstrip feature, since it's a UI indicator rather than a foundational (ie base) thing.